libevmasm/Instruction.cpp

```diff
@@ -339,6 +339,8 @@ static std::map<Instruction, InstructionInfo> const c_instructionInfo =
 	{Instruction::RETF,           {"RETF",            0,  0,   0,  true,       Tier::RetF}},
 	{Instruction::CALLF,          {"CALLF",           2,  0,   0,  true,       Tier::CallF}},
 	{Instruction::JUMPF,          {"JUMPF",           2,  0,   0,  true,       Tier::JumpF}},
+	{Instruction::SWAPN,          {"SWAPN",           1,  0,   0,  false,      Tier::VeryLow}},
+	{Instruction::DUPN,           {"DUPN",            1,  0,   0,  false,      Tier::VeryLow}},
```

Comment:
Since the numbers of inputs and outputs we have here for these instructions are meaningless, what do you think about setting them to something that can't be a real value?

```suggestion
	{Instruction::CALLF,          {"CALLF",           2, -1,  -1,  true,       Tier::CallF}},
	{Instruction::JUMPF,          {"JUMPF",           2, -1,  -1,  true,       Tier::JumpF}},
	{Instruction::SWAPN,          {"SWAPN",           1, -1,  -1,  false,      Tier::VeryLow}},
	{Instruction::DUPN,           {"DUPN",            1, -1,  -1,  false,      Tier::VeryLow}},
```

We could then have generic asserts against it in places that use the value (like `AssemblyItem::arguments()`).

Reply:
I was actually wondering the same - we can do that. But since it's a few preexisting cases as well, I'd do it separately.